src/components/pill/pill.component.js

```diff
@@ -2,12 +2,21 @@ import React from "react";
 import PropTypes from "prop-types";
 import StyledPill from "./pill.style";
 import Icon from "../icon";
-import { validProps } from "../../utils/ether/ether";
-import tagComponent from "../../utils/helpers/tags/tags";
+import { validProps } from "../../utils/ether";
+import tagComponent from "../../utils/helpers/tags";
 import IconButton from "../icon-button";
 
 class Pill extends React.Component {
-  static safeProps = ["onClick"];
+  static marginSpaceProps = ["m", "mt", "mr", "mb", "ml", "mx", "my"];
+
+  static getMarginSpacePropTypes() {
+    return Pill.marginSpaceProps.reduce((prev, curr) => {
+      prev[curr] = PropTypes.oneOfType([PropTypes.string, PropTypes.number]);
+      return prev;
+    }, {});
+  }
+
+  static safeProps = ["onClick", ...this.marginSpaceProps];
```

Comment:
```suggestion
static safeProps = ["onClick", ...propTypes.margin];
```

Reply:
Actually ignore me it looks like there is no propTypes for the subsets of `space`